README.md

```diff
@@ -91,30 +91,66 @@ Contribute adding resources or providing feedback through Pull Requests, Issues
 
 ## Toolkits
 
-- [rust-console](https://github.com/rust-console/gba) - GBA programming in Rust, plus tutorial.
-- [natu](https://github.com/exelotl/natu) - GBA programming in Nim (provides wrapper around libtonc, maxmod and more).
-- [gba-modern](https://github.com/JoaoBaptMG/gba-modern) - Write GBA games using modern C++.
-- [ZigGBA](https://github.com/wendigojaeger/ZigGBA) - WIP SDK for creating GBA games using Zig (Inspired by Tonc).
-- [Butano](https://github.com/GValiente/butano) - Modern C++ high level engine for the GBA.
-- [BPCore Engine](https://github.com/evanbowman/BPCore-Engine) - Create GBA applications with the Lua programming language
-- [GBA Dlang](https://github.com/redthing1/gba_dlang) - WIP SDK for GBA development using D, using LLVM toolchain. provides fully functional TONC, GBFS, Maxmod.
-- [dusk](https://github.com/redthing1/dusk) - Simple, lightweight, intuitive framework for GBA game development
-- [gbsenpai](https://github.com/asiekierka/gbsenpai) - GB Studio player GBA port with some enhancements.
+1. C and C++
+
+   - [gba-modern](https://github.com/JoaoBaptMG/gba-modern) - Write GBA games using modern C++.
+   - [dusk](https://github.com/redthing1/dusk) - Simple, lightweight, intuitive framework for GBA game development
+   - [Butano](https://github.com/GValiente/butano) - Modern C++ high level engine for the GBA.
+
+1. D
+
+   - [GBA Dlang](https://github.com/redthing1/gba_dlang) - WIP SDK for GBA development using D, using LLVM toolchain. provides fully functional TONC, GBFS, Maxmod.
```

Comment:
I don't think creating sublists of 1 (even 2) elements helps the visual clutter at all :/

Reply:
We have been talking about this on Discord. This is another suggestion about how to organize it: https://gist.github.com/exelotl/ae81b7906a24159008720ca37141885d

The main problem is that there isn't a clear separation between compilers, toolkits and libraries (specially seeing as some things such as Natu have everything in one!).